core/auth/auth.go

```diff
@@ -2451,49 +2457,49 @@ func (a *Auth) tokenDataForScopes(scopes []authorization.Scope) ([]string, strin
 	services := []string{}
 	for i, scope := range scopes {
 		scopeStrings[i] = scope.String()
-		if !authutils.ContainsString(services, scope.ServiceID) {
+		if !rokwireutils.ContainsString(services, scope.ServiceID) {
 			services = append(services, scope.ServiceID)
 		}
 	}
 
 	return services, strings.Join(scopeStrings, " ")
 }
 
-func (a *Auth) getStandardClaims(sub string, name string, email string, phone string, username string, aud string, orgID string, appID string, authType string, externalIDs map[string]string,
-	exp *int64, anonymous bool, authenticated bool, admin bool, system bool, service bool, firstParty bool, sessionID string, accessTokenExpPolicy *model.AccessTokenExpirationPolicy) tokenauth.Claims {
+func (a *Auth) getStandardClaims(sub string, name string, email string, phone string, username string, aud []string, orgID string, appID string, authType string, externalIDs map[string]string,
+	exp *time.Time, anonymous bool, authenticated bool, admin bool, system bool, service bool, firstParty bool, sessionID string, accessTokenExpPolicy *model.AccessTokenExpirationPolicy) tokenauth.Claims {
 	return tokenauth.Claims{
-		StandardClaims: jwt.StandardClaims{
-			Audience:  aud,
+		RegisteredClaims: jwt.RegisteredClaims{
+			Audience:  jwt.ClaimStrings(aud),
```

Comment:
Providing the audience as a list of strings ended up breaking backwards compatibility. Will revert this change back to a single comma-separated string for now.

Reply:
This change has been made here: https://github.com/rokmetro/core-building-block-fork/commit/fc548a901e7b0b6d227cefc79b2343b675323690